scripts/build_proc.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-MULTIARCH_DIR=/usr/lib/$(gcc -print-multiarch)
-FFTW_LIB=$MULTIARCH_DIR/libfftw3f.a
+# MULTIARCH_DIR=/usr/lib/$(gcc -print-multiarch)
+# FFTW_LIB=$MULTIARCH_DIR/libfftw3f.a
```

Comment:
Are the commented-out lines still needed?

Reply:
Yeah, we may need to revert to this at some point, so I'd rather keep these close at hand.